yt_dlp/extractor/cbc.py

```diff
@@ -676,7 +642,7 @@ def _real_extract(self, url):
         else:
             headers = {}
         m3u8_info = self._download_json(video_info['playSession']['url'], video_id, headers=headers)
-        m3u8_url = m3u8_info.get('url')
+        m3u8_url = re.sub(r'(manifestType=)[\w_-]+(&)', r'\1\2', m3u8_info.get('url'))
```

Comment:
this line should be moved below the error handling (and import `update_url_query` from utils)
```suggestion
        m3u8_url = update_url_query(m3u8_info['url'], {'manifestType': ''})
```